docs/learn/learn-guides-claims.md

```diff
@@ -0,0 +1,111 @@
+---
+id: learn-guides-claims
+title: Polkadot-JS Guides about Claims
+sidebar_label: Claim Guides
+description: Step-by-step guide on how to claim your tokens.
+keywords: [claims, tokens, kusama, polkadot]
+slug: ../learn-guides-claims
+---
+
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
+
+<MessageBox message="Polkadot-JS is for developers and power users only. If you need help using the
+[Polkadot-JS UI](../general/polkadotjs-ui.md), you can contact the
+[Polkadot Support Team](https://support.polkadot.network/support/home). For more user-friendly tools
+see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashboards-index) pages." />
+
+<DocCardList />
+
+If you participated in a previous DOT sale before 2020 and received your DOT allocation indicator
+tokens, you can now claim your DOT (and a proportional amount of KSM on the Kusama network). The
+claim process connects the address where your DOT indicators have been stored on Ethereum with a
+native {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} address and, if your ETH
+address is eligible, will pay the tokens to the
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} address.
+
+To do this, you must sign a message on Ethereum containing the address of your
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account. You can do this by using the
+Polkadot-JS UI [Claims app](https://polkadot.js.org/apps/#/claims). Ensure that you are connected to
+the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network (displayed in the
+upper-left-hand corner of the screen).
+
+:::warning Third-party claim processes
+
+Claiming using a third-party process can lead to losing your allocation; therefore, we cannot
+recommend using any third-party apps. Manually specifying your transaction data, as specified in our
+claims process below, is the only way to ensure you will receive your allocation.
+
+:::
+
+## Generate an Account
+
+You will need to generate a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account
+to claim {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. See the
+[available wallets and extensions](../general/wallets-and-extensions.md) for more information about
+wallets and browser extensions you can use to create an account. In terms of hardware wallet
+support, you can use the [Ledger](../general/ledger.md) devices or
+[Polkadot Vault](../general/polkadot-vault.md).
+
+### Address Conversion
+
+If you need to (most likely you don't), you can use
```

Comment:
> If you need to (most likely you don't)

The hexadecimal public key is needed for claiming DOT/KSM tokens. What is the intention behind this comment which implies this step is optional?

<img width="853" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/6010066c-15c2-4bbb-9c07-d8e340efd8a9">

Reply:
you do not need the tool to convert, you get it directly on pjs

Reply:
I did not need to use that tool to go through the process, that is why I wrote that. Maybe we can remove that section?

Reply:
Yeah, if Polkadot JS UI is doing it already, then this section can be deleted.

Reply:
done